### DISCUSSION TITLE
How to add custom styles to react-time-picker__wrapper class?

### DISCUSSION THREAD

**Discussion by agawankar** [wojtekmaj/react-time-picker#138] [Q&A] [answered]
It will be good if you provide wrapperClass as props to override wrapper class of the component

**Comment by wojtekmaj** [comment, marked as answer]
Pass `className="myClassName"` and the use nested styles:

```
.myClassName .react-time-picker__wrapper {
  outline: 2px solid red;
}
```

https://codesandbox.io/s/react-time-picker-custom-css-xwk0uc

You can also style it that way with e.g. styled-components:

https://codesandbox.io/s/react-time-picker-styled-components-gg7djb

**Reply by agawankar** [reply]
.myClassName .react-time-picker__wrapper {
  outline: 2px solid red;
}

This is not working in my case as I am using scss, 

so I tried with styled component and its working fine...

Thank you..!

**Reply by gabrielizalo** [reply]
It functioned flawlessly!  Thank you!
